scarlet/frame.py

```diff
@@ -241,55 +232,46 @@ def from_observations(
             else:
                 target_psf = target_psf_temp
 
-        # Margin in pixels
-        fat_pixel_size = (fat_psf_size / h).astype(int)
-        # Padding by the size of the psf
-        if fat_pixel_size % 2 != 0:
```

Comment:
Could you explain why?

Reply:
I've had the problem before. When Fourier transforming, we put the image in a grid with shape `fft_shape`, which is a fast shape to perform the fft. The issue is that it sometimes makes an even-shaped array odd-shaped or the other way around and it changes everything when convolving and bringing the array back to real space.

Reply:
OK, but we can't have stray centroid shifts in code! If parity is important for the FFT, it needs to be solved there, not everywhere we might need a FFT.

Reply:
Yup, but sometimes we use even sized array andd sometimes we use odd-sized. We need to make a choice so that the fft always reverts to the same. Anyway, I'll close it. This one is rather for me to figure out.

Reply:
Yeah, but it's important to figure out! Otherwise we keep running into issues with multi-resolution not performing well because of artificial shift that are impossible to diagnose.

Reply:
To be perfectly clear: I am seeing different shifts in your multiresolution test set with the old variant of the code snippet above as well as in the new, but none of them without shifts. So something is wrong.